sycl/source/detail/plugin.hpp

```diff
@@ -28,6 +28,8 @@ namespace detail {
 #ifdef XPTI_ENABLE_INSTRUMENTATION
 extern xpti::trace_event_data_t *GPICallEvent;
 extern xpti::trace_event_data_t *GPIArgCallEvent;
+extern uint8_t PiCallStreamID;
+extern uint8_t PiDebugCallStreamID;
```

Comment:
I don't see where these are being initialized (and again, no race condition?)

Reply:
Shouldn't be as they are initialized in initializePlugins() in pi.cpp, which should be a singleton.